### DISCUSSION TITLE
Zero visitors on Traffic Page

### DISCUSSION THREAD

**Discussion by Shriinivas** [community/community#23861] [New to GitHub]
<p>Hello Everyone!</p>
<p>For the past 5 days I see zero visitors to all of my repositories. But there have been changes in other stats like stargazers and watch list, so there were certainly some visitors.</p>
<p>What could be the reason for this issue on the traffic page?</p>
<p>Thank you.</p>

**Comment by huwb** [comment]
<p>Same for <a href="https://github.com/crest-ocean/crest" rel="nofollow noopener">https://github.com/crest-ocean/crest</a></p>
<p>Also for the last month or so the ratio of clones to unique cloners has been around 10:1, which also seems suspect.</p>

**Comment by PantelisElinas** [comment]
<p>Same problem for our library, zero traffic shown since the 21st of August: <a href="https://github.com/stellargraph/stellargraph" rel="nofollow noopener">https://github.com/stellargraph/stellargraph</a></p>
<p>P.</p>

**Comment by Ho-Ro** [comment]
<p>Same on my repos, I’ve just contacted github support about it.</p>
<p>Martin</p>

**Comment by Xottab-DUTY** [comment]
<p>The same problem for:</p>
<p><a href="https://github.com/OpenXRay" rel="nofollow noopener">https://github.com/OpenXRay</a></p>
<p><a href="https://github.com/xrOxygen" rel="nofollow noopener">https://github.com/xrOxygen</a></p>
<p>and my personal repos</p>
<p>GitHub Developer Support said that they are currently investigating the issue and will update as soon as there’s any news to share.</p>

**Comment by Ho-Ro** [comment]
<p>Got this email from support:</p>
<p><em>Hi there Martin,</em></p>
<p><em>Thanks for reaching out to GitHub Support, and sorry for the trouble.</em></p>
<p><em>We’re currently investigating similar reports regarding this issue, and I’ve added your report to our list.</em></p>
<p><em>While we can’t share a specific ETA just yet, we’ll be sure to update you as soon as there’s any news to share.</em></p>
<p><em>Thanks,</em></p>

**Comment by Ho-Ro** [comment, marked as answer]
<p>Now it’s working again.</p>

**Comment by Xottab-DUTY** [comment]
<p>I confirm. It works.</p>

**Comment by arogozhnikov** [comment]
<p>One year later, I see this immediate drop of visitors to zero for one of my projects.</p>
